tests/Integration/PrestaShopBundle/Routing/Converter/LegacyUrlConverterTest.php

```diff
@@ -305,32 +305,6 @@ public function testConverterByParameters(
         $this->assertSameUrl($expectedUrl, $convertedUrl);
     }
 
-    public function testTabParameter(): void
```

Comment:
Because this system of routes has been already nearly completely removed - https://github.com/PrestaShop/PrestaShop/pull/34290, it's a historic pre 1.5 routing that should not be used anymore - one less thing to maintain.